hack/gardener-extensions-down.sh

```diff
@@ -34,7 +34,7 @@ parse_flags() {
 
 parse_flags "$@"
 
-if ! kubectl config view --kubeconfig "$PATH_SEED_KUBECONFIG" --raw -o jsonpath='{.users[0].user.client-certificate-data}' | base64 --decode | openssl x509 -noout -checkend 10 2>/dev/null ;then 
+if ! kubectl config view --kubeconfig "$PATH_SEED_KUBECONFIG" --raw -o jsonpath='{.users[0].user.client-certificate-data}' | base64 --decode | openssl x509 -noout -checkend 300 2>/dev/null ;then 
```

Comment:
You only consider `kubeconfig`s with client certificates here, but what about tokens? It is possible to have a completely valid & working `kubeconfig` with a token, which will not pass this test.

Please adapt so that tokens are considered as well or at least not blocked.

Reply:
I am not sure about other options apart from client certificates and tokens. Maybe `gardenctl` based `kubeconfig`s should work as well.

Reply:
I'll make it so that if `.users[0].user.client-certificate-data` is missing the script does not get blocked. That's because I don't think I have the knowledge to handle the other cases